tests/Unit/QuerySortMergedTest.php

```
@@ -0,0 +1,253 @@
+use SlimStat\Utils\Query;
+
+class QuerySortMergedTest extends WpSlimstatTestCase
+{
+    private $wpdb;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->wpdb         = Mockery::mock('stdClass');
+        $this->wpdb->prefix = 'wp_';
```

PHPStan (2.1.46)

[error] Access to an undefined property Mockery\MockInterface::$prefix.
Learn more: https://phpstan.org/blog/solving-phpstan-access-to-undefined-property

(property.notFound)